src/components/views/EditMilestone.jsx

```diff
@@ -567,7 +567,7 @@ class EditMilestone extends Component {
       return Promise.reject();
     }
 
-    return authenticateUser(this.props.currentUser, true)
+    return authenticateIfPossible(this.props.currentUser, true)
```

Comment:
This change is not necessary, `authenticateIfPossible` is identical and should be removed from middleware.jsx